master/buildbot/db/buildrequests.py

```diff
@@ -397,31 +567,31 @@ def thd(conn):
         return self.db.pool.do(thd)
 
     def executeMergeBuildingRequests(self, conn, requests):
-            buildrequests_tbl = self.db.model.buildrequests
-            mergedrequests = [br.id for br in requests[1:]]
+        buildrequests_tbl = self.db.model.buildrequests
+        mergedrequests = [br.id for br in requests[1:]]
 
-            # we'll need to batch the brids into groups of 100, so that the
-            # parameter lists supported by the DBAPI aren't
-            iterator = iter(mergedrequests)
-            batch = list(itertools.islice(iterator, 100))
-            while len(batch) > 0:
-                q = sa.select([buildrequests_tbl.c.artifactbrid]) \
-                    .where(id == requests[0].id)
-                res = conn.execute(q)
-                row = res.fetchone()
-                # by default it will mark using artifact generated from merged brid
-                stmt2 = buildrequests_tbl.update() \
-                    .where(buildrequests_tbl.c.id.in_(mergedrequests)) \
-                    .values(artifactbrid=requests[0].id)\
-                    .values(mergebrid=requests[0].id)
+        # we'll need to batch the brids into groups of 100, so that the
```

Comment:
Next time try to keep format changes to a minimum or in another PR so it makes it easier to see that this is not part of the changes